src/bin/tikv-ctl.rs

```diff
@@ -1214,14 +1214,12 @@ fn get_module_type(module: &str) -> MODULE {
 }
 
 fn from_hex(key: &str) -> Vec<u8> {
```

Comment:
It seems it is possible for this function to fail. What is the reasoning behind using an `unwrap` and not returning a `Result`?

Reply:
I guess it's for simplicity.

Reply:
We can make from_hex returning Result<_> and unwraps this result if necessary.

Reply:
Maybe it'd better to move this along and create a follow up ticket with this change?

Reply:
@Hoverbear I like this idea. We can also leverage contributors to do this for a warm up!